docs/reference/modules/ROOT/pages/lucene.adoc

```diff
@@ -0,0 +1,78 @@
+= Lucene
```

Comment:
The ADoc template's changed a little while this branch has been in-flight, comments below

Reply:
:toc: stuff at the top no longer required

Reply:
Cool. Fixed.